atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/thrift/SlicePredicates.java

```diff
@@ -35,6 +35,14 @@ public static SlicePredicate latestVersionForColumn(byte[] columnName, long maxT
                 Limit.ONE);
     }
 
+    public static SlicePredicate rangeTombstoneForColumn(byte[] columnName, long maxTimestampExclusive) {
```

Comment:
It would be nice to see some testing around this, but short of actually running this on a test stack and looking with cqlsh at what gets written, I'm not sure how I'd go about it.

Reply:
The ETE tests should give us coverage for this